content/en/static_analysis/github_source_code_management.md

```diff
@@ -0,0 +1,65 @@
+---
+title: GitHub Source Code Management
+kind: documentation
+description: Feature overview and setup instructions for GitHub Source Code Management using Static Analysis.
+---
+
+## Feature overview
+
+Static Analysis can automatically flag rule violations in pull requests on GitHub. To configure the GitHub integration to include Static Analysis features, see [the setup instructions](#set-up-static-analysis-on-github).
```

Comment:
I think the URL anchor needs to be updated here since it didn't work in the preview